### PR TITLE
Allow MRCP Recognition-Timeout value passthrough to UniMRCP

### PR DESCRIPTION
**Pull request by Jared-Prime**
by setting the option flag properly

### CODE REVIEW

**Comment by Jared-Prime** [comment]
@sfgeorge @runningferret please review. we will need this if we ever want a dynamically set sensitivity level.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ vendor
 *.rbc
 tmp
 coverage
+tags
```
Comment (runningferret) [inline .gitignore:16]: Yiss

**Comment by runningferret** [comment]
Very nice cleanup :+1:

**Comment by Jared-Prime** [comment]
note that the static settings for Lumenvox are settable according to our [internal Ifbyphone documentation](https://github.com/Jared-Prime/apptastic/blob/c4e4fbe1fa42779efd3c0ba594bcdfcf933a63f9/doc/automatic_speech_recognition/tuning/synth_and_recog_options.md#lumenvox-settings).

**Comment by Jared-Prime** [comment]
@runningferret @sfgeorge this is now relevant again as a [dependency for this pull request](https://github.com/cloudvox/apptastic/pull/284)

File: lib/punchblock/translator/asterisk/component/mrcp_recog_prompt.rb [hunk 2]
```diff
@@ -9,6 +9,8 @@ module Component
         module MRCPRecogPrompt
           UniMRCPError = Class.new Punchblock::Error
 
+          DEFAULT_UNIMRCP_APP_OPTIONS = { uer: 1, b: 0, t: 5000, nit: -1, dit: -1}.freeze
```
Comment (Jared-Prime) [inline lib/punchblock/translator/asterisk/component/mrcp_recog_prompt.rb]: I should not have left in a hard coded setting for `t`

**Comment by sfgeorge** [comment]
This looks good to me, thank you!  @runningferret I defer to you, mate.

**Comment by runningferret** [comment]
Looks good to me :+1:

**Comment by sfgeorge** [comment]
@ajayanty Can you merge this for us?

**Comment by ajayanty** [comment]
This has been merged into cloudvox:develop-ifbyphone

**Comment by Jared-Prime** [comment]
This PR is misnamed; renaming it.

**Pull request merged**